### PR TITLE
Dgilman master

### PR DESCRIPTION
**Pull request by hippo91**
Just testing PR #4006 with some fixes dealing with pylint warnings

### CODE REVIEW

**Comment by hippo91** [comment]
@dgilman @Pierre-Sassoulas this just PR is just the same as #4006. I just fix pylint warnings by deleting unused `os` import and adding a pragma to disable locally `pylint` on the `__hash__` function.
As you can see the CI is still failing but only on Windows and with python3.8. 
2 tests are failing due to wrong column number. And both of them are dealing with an `elif` statement. It is very strange and has i do not have windows it is not really easy to investigate.
Any ideas?

File: pylint/testutils/output_line.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def __eq__(self, other):
             return self[:-1] == other[:-1]
         return NotImplemented  # pragma: no cover
 
-    def __hash__(self):
+    def __hash__(self):  # pylint:disable=invalid-hash-returned
         return None
```
Comment (cdce8p) [inline pylint/testutils/output_line.py:25]:
```suggestion
```
It should be fine to just remove it. From the python documentation:
```
A class that overrides __eq__() and does not define __hash__()
will have its __hash__() implicitly set to None.
```
https://docs.python.org/3/reference/datamodel.html#object.__hash__

File: tox.ini [hunk 2]
```diff
@@ -56,8 +56,6 @@ deps =
    https://github.com/PyCQA/astroid/tarball/master#egg=astroid
    coverage<5.0
    mccabe
-   # isort 5 is not compatible with Python 3.5
-   py35: isort>=4.2.5,<5
```
Comment (cdce8p) [inline tox.ini:60]: Shouldn't that be replaced by:
```py
isort
```

**Review by cdce8p — COMMENTED** [review]
I also noticed the error with Python 3.8 on Windows. Since it's just the column number, what do you think about not adding the test for 3.8 at the moment? That could still be done later, possibly together with adding the test for 3.9 on Windows.

The advantage would be that the master branch would finally be fixed and the MR backlog could be merged. With a possible release coming shortly there after.

File: pylint/testutils/output_line.py (hunk 1 above)
Comment (Pierre-Sassoulas) [reply pylint/testutils/output_line.py:25]: Yeah, I fixed that in #4077, can you do `git cherry-pick 293981fb130393dd310a75b601570907a4542387` ? We merged on master when the test were not passing and introduced additional errors.

File: tox.ini (hunk 2 above)
Comment (Pierre-Sassoulas) [reply tox.ini:60]: We'll remove python 3.5 in #4006

**Comment by Pierre-Sassoulas** [comment]
Hey I answer the two comments from @cdce8p too fast, I removed my comment :)

Regarding the column it's actually the contrary, above python 3.8 we should check the column, the value is better in 3.8.  See here: https://github.com/PyCQA/pylint/blob/master/pylint/testutils/output_line.py#L84. The question then is why are the test passing on Linux ?

**Comment by hippo91** [comment]
@Pierre-Sassoulas the fact that tests are not failing under linux is normal. The column returned by the ast node is the same as the one expected. 
My hypothesis is that the python ast elif node differs between windows and linux by not returning the same value.
However as i do not have a Windows os available i don't know how to verify this hypothesis. Any idea?

**Comment by Pierre-Sassoulas** [comment]
I think the clean way to fix that would be to add a configuration for ignoring the column in some functional tests, but maybe not testing anything under windows would be easier/faster especially since we want to release soon? [Here](https://github.com/PyCQA/pylint/blob/master/pylint/testutils/output_line.py#L84) we could add `        if not PY38_PLUS and not sys.platform == "win32":`

**Comment by cdce8p** [comment]
I believe this is just an issue with **AppVeyor**, although I don't know what exactly causes it.
I'm currently working on a PR to use Github Actions, since I noticed that Travis was kind of slow lately (a few hours if there is a backlog). Aside form a few unrelated issues with the test environment, all Windows tests (3.6 - 3.9) pass without issues.

**Comment by Pierre-Sassoulas** [comment]
Let's rebase this one on master once #4088 is merged.

**Comment by Pierre-Sassoulas** [comment]
Closed in favor of #4091

**Pull request closed without merging**